tensor2tensor/utils/multistep_optimizer_test.py

```diff
@@ -0,0 +1,123 @@
+# coding=utf-8
```

Comment:
Thank you for adding this test, but I think the test should be a bit different and hopefully simpler:

Compare 2 things:
1. AdamOptimizer with batch size 32 for 1 step
2. MultistepAdamOptimizer with batch size 8 for 4 steps with n=4

We should see that the updates are identical (i.e. the variables end up in the same place)

Reply:
Hm am I not doing something like this? Just that I don't compare batch sizes but number of updates. For example, I compare the variables after

1. Adam: t steps with averaged gradients over n steps
2. MultistepAdam: t*n steps

for n=1..4 and t=1..3. Adam is implemented in numpy to avoid introducing dependencies from this test class to other parts of the TF code (like in the original Adam test)

Reply:
Alright np I'll change it